container/image_test.py

```diff
@@ -830,6 +832,30 @@ def test_nodejs_image_args(self):
         'arg1',
       ])
 
+def load_stamp_info():
+  stamp_file = TestData("stamp_info_file.txt")
+  with open(stamp_file) as stamp_fp:
+    for line in stamp_fp:
+      # The first column in each line in the stamp file is the key
+      # and the second column is the corresponding value.
+      split_line = line.strip().split()
+      if len(split_line) == 0:
+        # Skip blank lines.
+        continue
+      key = ""
```

Comment:
or
```
key = split_line[0]
value = ""
if len(split_line) > 1
  value = " ".join(split_line[1:])
```
Both are fine to me. Your current implementation is better in readability.